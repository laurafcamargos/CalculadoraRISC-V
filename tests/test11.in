10
+
5
*
3
-
4
/
2
/
0
+
2
u
u
u
u
u
abc
-5
*
2
/
-3
u
x
/
3.14
/
-0.1
/
3
f
