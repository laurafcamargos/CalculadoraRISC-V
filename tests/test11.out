Digite um número: Digite a operação (+, -, *, /, u, f): Digite um número: Resultado: 15
Digite a operação (+, -, *, /, u, f): Digite um número: Resultado: 45
Digite a operação (+, -, *, /, u, f): Digite um número: Resultado: 41
Digite a operação (+, -, *, /, u, f): Digite um número: Resultado: 20
Digite a operação (+, -, *, /, u, f): Digite um número: Erro: divisão por zero
Digite a operação (+, -, *, /, u, f): Digite um número: Resultado: 22
Digite a operação (+, -, *, /, u, f): Operação desfeita. Resultado anterior: 20
Digite a operação (+, -, *, /, u, f): Operação desfeita. Resultado anterior: 41
Digite a operação (+, -, *, /, u, f): Operação desfeita. Resultado anterior: 45
Digite a operação (+, -, *, /, u, f): Operação desfeita. Resultado anterior: 15
Digite a operação (+, -, *, /, u, f): Operação desfeita. Nenhum resultado salvo.
Digite um número: Erro: valor não é um número inteiro válido
Digite um número: Digite a operação (+, -, *, /, u, f): Digite um número: Resultado: -10
Digite a operação (+, -, *, /, u, f): Digite um número: Resultado: 3
Digite a operação (+, -, *, /, u, f): Operação desfeita. Resultado anterior: -10
Digite a operação (+, -, *, /, u, f): Erro: operação inválida
Digite a operação (+, -, *, /, u, f): Digite um número: Erro: valor não é um número inteiro válido
Digite um número: Erro: valor não é um número inteiro válido
Digite um número: Erro: valor não é um número inteiro válido
Digite um número: Erro: valor não é um número inteiro válido
Digite um número: Resultado: -3
Digite a operação (+, -, *, /, u, f): 
